pkg/templatecache/file_manager.go

```diff
@@ -10,7 +10,7 @@ import (
 	"path/filepath"
 	"strings"
 
-	"github.com/massdriver-cloud/liquid"
+	"github.com/osteele/liquid"
```

Comment:
Swapped back to the official library since they merged my PR, however there were more whitespace issues addressed by another PR on the upstream which I merged into our internal fork. We handle the redirect in go.mod now.